aapcs64-morello/aapcs64-morello.rst

```diff
@@ -496,7 +496,25 @@ The A64 and C64 branch instructions are unable to reach every destination in the
 Parameter Passing
 -----------------
 
-The base standard provides for passing arguments in general-purpose registers (r0-r7), SIMD/floating-point registers (v0-v7) and on the stack. For subroutines that take a small number of small parameters, only registers are used.
+The base standard provides for passing arguments in general-purpose registers (r0-r7), SIMD/floating-point registers (v0-v7) and in memory. For subroutines that take a small number of small parameters, only registers are used.
+
+Memory-passed parameters
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We define an argument as being memory-passed if it copied to memory as part of Stage C from `Parameter Passing Rules`_. The address of a memory-passed argument is the address of the memory location where the argument was copied to as part of Stage C from `Parameter Passing Rules`_.
+
+Arguments Capability
+^^^^^^^^^^^^^^^^^^^^
+
+In AAPCS64-cap if there are any memory-passed parameters the Arguments Memory Area is allocated.
+
+The Memory Arguments Length (MAL) is the number of bytes required for memory-passed arguments. This can be obtained by subtracting the final and initial values of NMAA as part of Stage C from `Parameter Passing Rules`_.
+
+The Rounded Memory Arguments Length (RMAL) is the least representable capability length greater or equal to MAL.
+
+The size of the Arguments Memory Area is equal to RNML. The minimal alignment of the Arguments Memory Area is 16 bytes. The alignment of the Arguments Memory Area is large enough to allow the bounds of the Arguments Memory Area to be exactly representable by a capability.
```

Comment:
I can't find the defiinition of RNML, could it be RMAL?

Reply:
Thanks! Yes, it should have been RMAL.